powerauth-client-model/src/main/java/com/wultra/security/powerauth/client/model/request/OperationCreateRequest.java

```diff
@@ -38,5 +38,6 @@ public class OperationCreateRequest {
     private String externalId;
     private final Map<String, String> parameters = new LinkedHashMap<>();
     private Boolean proximityCheckEnabled;
+    private String activationId;
```

Comment:
1) May we distinguish for swagger what is optional and what is required, please?
2) Where it is validated?